src/backend/distributed/utils/metadata_cache.c

```diff
@@ -3415,3 +3419,26 @@ DistNodeMetadata(void)
 
 	return metadata;
 }
+
+
+Datum
+role_exists(PG_FUNCTION_ARGS)
+{
+	Name roleName = PG_GETARG_NAME(0);
+	bool roleExists = SearchSysCacheExists1(AUTHNAME, NameGetDatum(roleName));
+
+	PG_RETURN_BOOL(roleExists);
+}
+
+
+Datum
+authinfo_valid(PG_FUNCTION_ARGS)
+{
+	ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					errmsg("catalog table unsupported"),
+					errdetail("Citus Community Edition does not support the use of per-"
+							  "host/user authentication options."),
```

Comment:
`per host/` is probably not relevant here

Reply:
Do we want to maybe let Community users at least specify a password here? Using `pgpass` is still kind of annoying and requires filesystem access. I think I wrote this error message with the idea that we might let them set at least a global password.